functions/exercises.py

```diff
@@ -167,14 +166,17 @@ def get_unique_item_list(original_list):
 def check_prime(number):
     """
     Check if given number is a prime number
-
     :param number:
     :return:
     """
-    # INSERT YOUR CODE HERE
-    #
-    # Run this command to test your implementation:
-    #
+    if number <= 1:
+        return False
+
+    for n in range(2,int(math.ceil(number**0.5))):

```

Comment:
```suggestion
    for n in range(2,math.ceil(number**0.5)):
```
`math.ceil` is supposed to give you an integer already right?

Can you explain a bit your line of thought?
Why do we need `number**0.5`? 
and why don't you use `math.sqrt` since you have already imported that module?

Reply:
math.ceil is indeed already returning an int so no need for the int cast.

Using the exponentiation operator ** is an easy way of getting the square root of a number. This operator raises the first operand to the power of the second operand.

To get square root, you need to use 0.5 as your second operand

also and following your suggestion of using math.sqrt will update to:

from math import sqrt

# Number to be checked for prime

flag = 0

if(number > 1):
	for k in range(2, int(sqrt(number)) + 1):
		if (number % k == 0):
			flag = 1
		break
	if (flag == 0):
		print(number," is a Prime Number!")
	else:
		print(number," is Not a Prime Number!")
else:
	print(number," is Not a Prime Number!")